content/conda.md

```diff
@@ -1,137 +1,256 @@
+    :language: yaml
+    :::
+  ::::
+
+  ::::{tab} requirements.txt
+    :::{literalinclude} software/requirements.txt
+    :::
+  ::::
+:::::
+
+:::{admonition} If you have a tool/method that works for you, keep using it
+If you are used to installing Conda packages and know what to do with the
+above files, please follow your own preferred installation
+method.
+
+**If you are new to Conda or unsure** how to create isolated environments from
+files above, please follow the instructions below where we do this with the help
+of Miniforge.
+
+If you already use **Anaconda** to manage your software environments, please
+continue using it. In this case you can skip {ref}`miniforge` and go directly to
+{ref}`conda-environment`.
+:::
+
+
+:::{admonition} Why do we recommend Miniforge and how does it relate to Conda?
+:class: dropdown
+
+There are many choices and we try to suggest a good compromise
+There are very many ways to install packages with pros and cons and
+in addition there are several operating systems with their own quirks. This
+can be a huge challenge for beginners to navigate. It can also difficult for
+instructors to give recommendations for something which will work everywhere
+and which everybody will like.
+
+Below we will recommend **Miniforge** since it is free, open source, general,
+available on all operating systems, and provides a good basis for reproducible
+environments. However, it does not provide a graphical user interface during
+installation.
+
+But Miniforge is only one option. Unfortunately there are many options and a
+lot of jargon.  Here is a crash course:
+
+- **Conda** is a package manager: it allows distributing and
+  installing packages, and is designed for complex scientific
+  code.
+- **Mamba** is a re-implementation of Conda to be much faster with
+  resolving dependencies and installing things.
+- An **environment** is a self-contained collections of packages
+  which can be installed separately from others.  They are used so
+  each project can install what it needs without affecting others.
+- **Anaconda** is a commercial distribution of Python+Conda+many
+  packages that all work together.  It used to be freely usable for
+  research, but since ~2023-2024 it's more limited.  Thus, we don't
+  recommend it (even though it has a nice graphical user interface).
+- **conda-forge** is another channel of distributing packages that
+  is maintained by the community, and thus can be used by anyone.
+  (Anaconda's parent company also hosts conda-forge packages)
+- **Miniforge** is a distribution of conda pre-configured for
+  conda-forge.  It operates via the command line.
+- **Miniconda** is a distribution of conda pre-configured to use
+  the Anaconda channels.
+:::
+
+
+(miniforge)=
+
+## Installing Miniforge
+
+Download the latest release from the [Miniforge download
+overview](https://conda-forge.org/download/).  This installs the base, and from
+here other packages can be installed.
+
```

Comment:
This download page (which I did not know) is much easier to understand than the README.md at https://github.com/conda-forge/miniforge. 

Also, Chrome does allow the download from here.

Still, on Windows, Microsoft "Defender" "Smart" screen opposes  the installation, but it still works via powershell.

Reply:
Yeah I found it today for the first time :-)